Mathlib/Algebra/Order/Field/Basic.lean

```diff
@@ -339,13 +340,14 @@ theorem one_le_inv_iff : 1 ≤ a⁻¹ ↔ 0 < a ∧ a ≤ 1 :=
 -/
 
 
-@[mono]
+@[mono, gcongr]
 theorem div_le_div_of_le (hc : 0 ≤ c) (h : a ≤ b) : a / c ≤ b / c := by
   rw [div_eq_mul_one_div a c, div_eq_mul_one_div b c]
   exact mul_le_mul_of_nonneg_right h (one_div_nonneg.2 hc)
 #align div_le_div_of_le div_le_div_of_le
 
 -- Not a `mono` lemma b/c `div_le_div` is strictly more general
+@[gcongr]
 theorem div_le_div_of_le_left (ha : 0 ≤ a) (hc : 0 < c) (h : c ≤ b) : a / b ≤ a / c := by
```

Comment:
Does the same concern not apply to `gcongr`?

Reply:
It doesn't apply; `gcongr` wants left/right lemmas even if the "both arguments" lemma implies both of them.